From ef6ee5b1b3023dc792c48a3b26000711e412d113 Mon Sep 17 00:00:00 2001
From: Hariprasad Kelam <hkelam@marvell.com>
Date: Mon, 12 Oct 2020 15:40:38 +0530
Subject: [PATCH 0899/1921] octeontx2-af: cn10k: Add CN10K PTP subsytem ID

Adds CN10K PTP subsystem and device IDs to ptp driver id table.

Change-Id: Ie77bdb9a99510a7f4a5e63053604b45cefccc85c
Signed-off-by: Geetha sowjanya <gakula@marvell.com>
Signed-off-by: Sunil Goutham <sgoutham@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/ptp.c | 16 ++++++++++------
 1 file changed, 10 insertions(+), 6 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/ptp.c b/drivers/net/ethernet/marvell/octeontx2/af/ptp.c
index 8d1a0ec7ad9f..ad4ac5fae8e9 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/ptp.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/ptp.c
@@ -16,12 +16,13 @@
 
 #define DRV_NAME	"Marvell PTP Driver"
 
-#define PCI_DEVID_OCTEONTX2_PTP		0xA00C
-#define PCI_SUBSYS_DEVID_OCTX2_98xx_PTP	0xB100
-#define PCI_SUBSYS_DEVID_OCTX2_96XX_PTP	0xB200
-#define PCI_SUBSYS_DEVID_OCTX2_95XX_PTP	0xB300
-#define PCI_SUBSYS_DEVID_OCTX2_LOKI_PTP	0xB400
-#define PCI_DEVID_OCTEONTX2_RST		0xA085
+#define PCI_DEVID_OCTEONTX2_PTP			0xA00C
+#define PCI_SUBSYS_DEVID_OCTX2_98xx_PTP		0xB100
+#define PCI_SUBSYS_DEVID_OCTX2_96XX_PTP		0xB200
+#define PCI_SUBSYS_DEVID_OCTX2_95XX_PTP		0xB300
+#define PCI_SUBSYS_DEVID_OCTX2_LOKI_PTP		0xB400
+#define PCI_SUBSYS_DEVID_CN10K_A_PTP		0xB900
+#define PCI_DEVID_OCTEONTX2_RST			0xA085
 
 #define PCI_PTP_BAR_NO	0
 #define PCI_RST_BAR_NO	0
@@ -234,6 +235,9 @@ static const struct pci_device_id ptp_id_table[] = {
 	{ PCI_DEVICE_SUB(PCI_VENDOR_ID_CAVIUM, PCI_DEVID_OCTEONTX2_PTP,
 			 PCI_VENDOR_ID_CAVIUM,
 			 PCI_SUBSYS_DEVID_OCTX2_LOKI_PTP) },
+	{ PCI_DEVICE_SUB(PCI_VENDOR_ID_CAVIUM, PCI_DEVID_OCTEONTX2_PTP,
+			 PCI_VENDOR_ID_CAVIUM,
+			 PCI_SUBSYS_DEVID_CN10K_A_PTP) },
 	{ 0, }
 };
 
-- 
2.31.1

